sql-parser/src/main/java/io/crate/sql/SqlFormatter.java

```diff
@@ -679,6 +683,30 @@ protected Void visitLongLiteral(LongLiteral node, Integer indent) {
             return null;
         }
 
+        @Override
+        protected Void visitIntegerLiteral(IntegerLiteral node, Integer context) {
+            builder.append(String.format(Locale.ENGLISH, "%d", node.getValue()));
```

Comment:
Could probably also use the `<TypeName>.toString(node.getValue())` variants here - they've less overhead.